pagopa/io-app
# chore(IT Wallet): [SIW-535] Refactor card component

## Short description
This PR proses a refactor to the `PidCredential.tsx` card component to generalize it for any credential.

## List of changes proposed in this pull request
- Renames the component to `ItwCredentialCard`;
- Adds `backgroundImage` prop;
- Simplifies the styling;  
- Replaces typography components with the new design system.

## How to test
Test the activation flow and check for any regression. Also check the preview in the wallet section after obtaining a PID.
